app/templates/src/main/java/package/security/_SecurityUtils.java

```diff
@@ -36,6 +36,22 @@ public static String getCurrentLogin() {
     }
 
     /**
+     * Get the UserDetails associated with the Current User.
+     */
+    public static UserDetails getCurrentLoginId() {
```

Comment:
The name of the method isn't good, it doesn't get the login id, but a user details